helm/aws-load-balancer-controller/README.md

```diff
@@ -18,6 +18,8 @@ AWS Load Balancer controller manages the following AWS resources
 
 ## Security updates
 **Note**: Deployed chart does not receive security updates automatically. You need to manually upgrade to a newer chart.
+#### Node isolation
+For the best practice, it is recommended to isolate the controller deployment pod to a specific node. The helm chart provides parameters ```nodeSelector```, ```tolerations``` and ```affinity``` to configure node isolation. For more information, please refer to the guidance [here](https://aws.github.io/aws-eks-best-practices/security/docs/multitenancy/#isolating-tenant-workloads-to-specific-nodes).
```

Comment:
Lets reword the first sentence as:
As a security best practice, we recommend isolating the controller deployment pods to specific node groups which run critical components.

Reply:
Amended, thanks